sdk/communication/tests.yml

```diff
@@ -0,0 +1,30 @@
+trigger: none
+
+parameters:
+  - name: TestPackagesEnabled # by overriding this parameter, developers can control what tests to run
+    displayName: Test Enabled
+    type: string
+    default: all # run live/sample tests for all packages by default
+    values:
+      - all # run live/sample tests for all packages
+      - chat # run live/sample tests for chat package only
+      - identity # run live/sample tests for identity package only
+      - phonenumbers # run live/sample tests for phonenumber package only
+      - sms # run live/sample tests for sms package only
+      - servercalling # run live/sample tests for servercalling package only
+
+extends:
+  template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
+  parameters:
+    ServiceDirectory: communication
+    CloudConfig:
+      Public:
+        SubscriptionConfigurations:
```

Comment:
This part will need to be updated too. Refer to the other tests.yml in communication package.

Reply:
updated.